Cineaste/ViewController/MovieDetail/MovieDetailViewController.swift

```diff
@@ -160,6 +169,18 @@ class MovieDetailViewController: UIViewController {
 
     // MARK: - Private
 
+    fileprivate func generateMovieURL() -> URL? {
+        var movieUrl = Config.Backend.shareMovieUrl
+
+        if let movie = storedMovie {
+            movieUrl += "\(movie.id)"
+        } else if let movie = movie {
+            movieUrl += "\(movie.id)"
+        }
```

Comment:
We could add an else branch with a precondition Failure, as this should never happen

`preconditionFailure("Either movie or storedMovie must be set to generate share movie url")`